plugins/de.ovgu.featureide.fm.core/src/de/ovgu/featureide/fm/core/base/IConstraint.java

```diff
@@ -314,4 +314,18 @@
 	 */
 	void setFalseOptionalFeatures(Iterable<IFeature> foFeatures);
 
+	/**
+	 * Set the description
+	 * @param description
+	 */
+	void setDescription(CharSequence description);
```

Comment:
Why CharSequence and not String?

Reply:
I've changed it to String, because internally we were already using String and the CharSequence Interface has undefined behaviour for the equals method.